### ISSUE TITLE
Vagrantfile install unsuccessful

### ISSUE THREAD

**Issue by vrracing** [elmsln/elmsln#989]
Tried following the vagrant install route but no joy.  Going to http://online.elmsln.local/ results in...

![image](https://cloud.githubusercontent.com/assets/2730237/16366741/51c42010-3be1-11e6-8e10-2fdfe5e736ac.png)

MacOSX 10.11.5
VirtualBox Version 5.0.22 r108108
Vagrant 1.8.4

There are errors when the `vagrant up` runs but don't know which are expected.  Console output is attached.
[vagrantUp.txt](https://github.com/elmsln/elmsln/files/333998/vagrantUp.txt)

**Comment by btopro** [comment]
it claims that Starting mysqld wasn't restarted correctly right before install yet did fine later. Could you do vagrant destroy then vagrant up again just to make sure it wasn't a fluke? Other then that 1 line not restarting mysql appropriately it worked (yeah, I know that means everything failed, lol)

**Comment by vrracing** [comment]
Thx @btopro.  I did that three times before posting the question just to make sure it wasn't anything on my side.

So, can I do the restarting mysql thing to make it work?

**Comment by btopro** [comment]
no cause it has to restart itself during spin up... I dono I'm not experiencing this issue and am on vacation currently. I'll take a look when I get back but there's no reason mysql shouldn't be restarting itself at this point int he operation, especially as it does restart successfully later on. did u try playing with http://online.do1.demos.elmsln.org/user using staff/staff

**Comment by mmilutinovic1313** [comment]
@vrracing & @btopro  I have replicated this issue and am checking it out now. Quick question for you (@vrracing) what is your vagrant version and what is your virtual box version?

**Comment by heyMP** [comment]
I believe this is do to vagrant installing installing PHP 5.5. Travis.ci is installing 5.6 and the install completes without error.

`vagrant up` errors whenever it attempts to restart apache with `sudo service apache2 restart`.  Since it is  PHP 5.5 this command will not work.

I'm trying to find out where the PHP version is specified in vagrant.

**Comment by mmilutinovic1313** [comment]
I'm running a travis build in 5.5 on my local copy to see if that is what the error is being caused by.

**Comment by mmilutinovic1313** [comment]
The version of PHP running in my vagrant after doing `vagrant ssh` and then a `php --version` is 5.5.37. We think that the issue may trace back to the `elmsln/centos-6.7` box and where it sets the php version variable. Testing in travis allows us to set a custom PHP version on install. I'm working out the kinks in my travis.yml file and then will update you all on the status.

**Comment by heyMP** [comment]
Yeah so we're dumb.  PHP version has nothing to do with it.  I don't know
what that `service apache2 restart` is being called.

**Comment by btopro** [comment]
server apache2 restart is in a vagrant clean up file and is unrelated. The issue happens very early on in the build process where mysql doesn't correctly restart (agian, just in @vrracing setup cause I haven't been able to replicate thus far). @vrracing how much RAM do you have by chance?

**Comment by btopro** [comment]
@vrracing ok, I think we traced the vagrant issue down. can you pull master and then vagrant destroy / vagrant up again please

**Comment by mmilutinovic1313** [comment]
@vrracing just verified the last commit by @btopro worked for getting this to work with `vagrant destroy` and `vagrant up`. Try it again and let me know if it works out for you as well! :)

**Comment by vrracing** [comment]
By George I think you've got it!  That fixed it.  I have a running Vagrant now.  Thanks a bunch!
